options/verify.go

```diff
@@ -26,6 +28,10 @@ type VerifyOptions struct {
 	CAPaths              []string
 }
 
+var RequiredVerifyFlags = []string{
+	"policy",
```

Comment:
I think the public key will also be required.

Reply:
So for verify *either* `--publickey` or `--policy-ca` can be specified: https://github.com/in-toto/witness/blob/main/cmd/verify.go#L59-L61

Reply:
https://pkg.go.dev/github.com/spf13/cobra#Command.MarkFlagsOneRequired?

Reply:
Ah - thank you for this @adityasaky I missed it!